sample-templates/linux-grub-zvol.conf

```diff
@@ -1,8 +1,12 @@
+# Uncomment to enable the graphical console
+#graphics="yes"
+#xhci_mouse="yes"
+
 loader="grub"
-cpu=1
-memory=512M
+cpu="1"
+memory="1G"
 network0_type="virtio-net"
 network0_switch="public"
+disk0_type="virtio-blk"
 disk0_name="disk0"
 disk0_dev="sparse-zvol"
-disk0_type="virtio-blk"
```

Comment:
Is this sufficient w/o addtional grub parameters?

Reply:
What I would do:
* Remove the `-uefi` suffix from the Linux ones to make it default
* Don't remove the GRUB examples, but add  fat big warning, that it might not be complete/work for your distro, you are encouraged to use UEFI